src/util/StringHash.cc

```diff
@@ -89,7 +97,7 @@ namespace lsst::qserv::util {
 /// @return a hexadecimal representation of the MD5 hash of the input buffer
 /// 128 bits -> 16 bytes -> 32 hex digits
 std::string StringHash::getMd5Hex(char const* buffer, int bufferSize) {
-    return wrapHashHex<MD5, MD5_DIGEST_LENGTH>(buffer, bufferSize);
+    return wrapHashHex<WMD5, MD5_DIGEST_LENGTH>(buffer, bufferSize);
```

Comment:
Though I 100% agree with the fix as a short-term solution, the very presence of the method `StringHash::getMd5Hex(` is a design mistake in Qserv. The only use of the method is for generating random names of the temporary tables in:
```
void UserQuerySelect::submit() {
    ...
    TmpTableName ttn(_queryId, _qSession->getOriginal());
```
Since `queryId` is unique in each Qserv deployment, it's easy to make unique temporary table names by simply naming those as:
```
"temporary_" + std::to_string(_queryId)
```
Instead of dragging the whole `openssl/MD5` machinery for that.

I'm not proposing to make this change now. Your change was the right thing to do in the scope of the current PR. We should probably do the right thing in a separate PR.

Reply:
Thanks, Igor -- good suggestion.  Would gladly tackle on a separate ticket.